samples/msal-node-samples/nextjs-rsc-ts/.gitignore

```diff
@@ -0,0 +1,35 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# next.js
+/.next/
+/out/
+
+# production
+/build
+
+# misc
+.DS_Store
+*.pem
+
+# debug
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# local env files
+.env*.local
+
+# vercel
+.vercel
+
+# typescript
+*.tsbuildinfo
+next-env.d.ts
```

Comment:
```suggestion
next-env.d.ts

# vscode settings
.vscode
```

Reply:
Ok, I tried and ran into the same problem. I'd prefer the least amount of friction for devs, so let's keep it then : )